pkgs/development/libraries/libbfd/default.nix

```diff
@@ -9,12 +9,11 @@ stdenv.mkDerivation {
 
   outputs = [ "out" "dev" ];
 
+  # Can't use fetchpatch due to interdependency on binutils:
+  #   https://github.com/NixOS/nixpkgs/issues/82792
   patches = binutils-unwrapped.patches ++ [
     ../../tools/misc/binutils/build-components-separately.patch
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/mxe/mxe/e1d4c144ee1994f70f86cf7fd8168fe69bd629c6/src/bfd-1-disable-subdir-doc.patch";
-      sha256 = "0pzb3i74d1r7lhjan376h59a7kirw15j7swwm8pz3zy9lkdqkj6q";
-    })
+    ./bfd-1-disable-subdir-doc.patch
```

Comment:
Hmm so how did this work before? Caching and Substitution?

Reply:
I see something in the issue. Checking.

Reply:
AFAIU `libbfd` is not used in non-all-target binutils:

```nix
, withAllTargets ? false, libbfd, libopcodes
...
  postFixup = lib.optionalString (enableShared && withAllTargets) ''
    rm "$out"/lib/lib{bfd,opcodes}-${version}.so
    ln -s '${lib.getLib libbfd}/lib/libbfd-${version}.so' "$out/lib/"
    ln -s '${lib.getLib libopcodes}/lib/libopcodes-${version}.so' "$out/lib/"
  '';
```

As to why exactly `binutils-all-targets` is affected by `libbfd` i'm not sure. It's somehow related to the `src` hack in `binutils`:

```
  # HACK: Ensure that we preserve source from bootstrap binutils to not rebuild LLVM
  src = stdenv.__bootPackages.binutils-unwrapped.src
    or srcs.${targetPlatform.system}
    or srcs.normal;
```